### ISSUE TITLE
Rename `sizing` to `instance_group` in helm values

### ISSUE THREAD

**Issue by mook-as** [cloudfoundry-incubator/kubecf#1303]
In #1215 we added `sizing.*.affinity`, and in #1295 we expanded that with `sizing.*.tolerations`; those things are not really _sizing_ per se, but instance group settings.  Looking at everything else under the `sizing` keys, we _should_ be able to just rename it `instance_groups.*` instead.  That would make the configuration more understandable.

I'm filing this issue to gather feedback on if that makes sense, and _if_ we agree we could actually schedule this.

/cc @viovanov @jandubois as people who'd likely have opinions.

**Comment by jandubois** [comment]
> /cc @viovanov @jandubois as people who'd likely have opinions.

How did you know I would have opinions? 😀 

I agree that `sizing` is not a great name; one reason we choose it was for backwards compatibility with `SCF`, but that was already kind of broken by the gratuitous renaming of `sizing.*.count` to `sizing.*.instances`.

I don't really like the word `instance_groups`, as it is long and unwieldy, and also relates to the bosh-in-vms world, where it refers to a group of identically configured VMs.

So I think we want to use a kube-native term for this, that will also cover the non-bosh-release based parts of kubecf, like bits, and eirini, and whatever native components we include from cf4k8s.

The best term I can think of right now that includes all the different concepts we have combined here is `scheduling`: All the settings are trying to influence how kube will schedule pods on nodes. But please keep suggestions coming!

I would prefer to delay renaming the key until we have reviewed *all* of our settings and create some guideline for how things should be restructured in consistent manner. I don't want users to have to adjust some settings on every other release; we should do it all at once.

I've already committed to create a separate issue for this in the "followup todo" section of #1202  Maybe I should create a RFD for this because the first phase would be discussion/brainstorming until we all agree on the future layout.

**Comment by viovanov** [comment]
It would be nice to have settings mean something and be "more great".
I would like to see some feedback from users that have read the docs and have deployed kubecf before starting a larger discussion.
This might be a non-issue.
